packages/charts/src/components/ScatterChart/ScatterChart.tsx

```diff
@@ -0,0 +1,212 @@
+  opacity?: number;
+  /**
+   * Defines axis of measures
+   */
+  axis: 'x' | 'y' | 'z';
+}
+
+interface ScatterDataObject {
+  label?: string;
+  data?: any[];
+  color?: CSSProperties['color'];
+}
+
+export interface ScatterChartProps extends IChartBaseProps {
+  dataset?: ScatterDataObject[];
+  /**
+   * An array of config objects. Each object is defining one line in the chart.
+   *
+   * <h4>Required properties</h4>
+   * - `accessor`: string containing the path to the dataset key this line should display. Supports object structures by using <code>'parent.child'</code>.
+   *   Can also be a getter.
+   *
+   */
+  measures?: MeasureConfig[];
+}
+
+const measureDefaults = {
+  formatter: defaultFormatter
+};
+
+/**
+ * <code>import { ScatterChart } from '@ui5/webcomponents-react-charts/lib/ScatterChart';</code>
+ */
+const ScatterChart: FC<ScatterChartProps> = forwardRef((props: ScatterChartProps, ref: Ref<HTMLDivElement>) => {
+  const {
+    dataset,
+    loading,
+    noLegend = false,
+    noAnimation = false,
+    onDataPointClick,
+    onLegendClick,
+    style,
+    className,
+    tooltip,
+    slot
+  } = props;
+
+  const chartConfig = useMemo(() => {
+    return {
+      yAxisVisible: false,
+      xAxisVisible: true,
+      gridStroke: ThemingParameters.sapList_BorderColor,
+      gridHorizontal: true,
+      gridVertical: false,
+      legendPosition: 'bottom',
+      legendHorizontalAlign: 'left',
+      zoomingTool: false,
+      resizeDebounce: 250,
+      ...props.chartConfig
+    };
+  }, [props.chartConfig]);
+
+  const { measures } = usePrepareDimensionsAndMeasures([], props.measures, {}, measureDefaults);
+
+  const tooltipValueFormatter = useTooltipFormatter(measures);
+  const chartRef = useConsolidatedRef<any>(ref);
+  const onItemLegendClick = useLegendItemClick(onLegendClick);
+
+  const onDataPointClickInternal = useCallback(
+    (payload, eventOrIndex) => {
+      if (eventOrIndex.dataKey && onDataPointClick) {
+        onDataPointClick(
+          enrichEventWithDetails(
+            {},
+            {
+              value: eventOrIndex.value,
+              dataKey: eventOrIndex.dataKey,
+              dataIndex: eventOrIndex.index,
+              payload: eventOrIndex.payload
+            }
+          )
+        );
+      }
+    },
+    [onDataPointClick]
+  );
+  const isBigDataSet = dataset?.length > 30 ?? false;
+
+  const xMeasure = measures.find(({ axis }) => axis === 'x');
+  const yMeasure = measures.find(({ axis }) => axis === 'y');
+  const zMeasure = measures.find(({ axis }) => axis === 'z');
+
+  const [yAxisWidth, legendPosition] = useLongestYAxisLabel(dataset?.[0].data, [yMeasure]);
+  const xAxisHeights = useObserveXAxisHeights(chartRef, 1);
+  const marginChart = useChartMargin(chartConfig.margin, chartConfig.zoomingTool);
+
+  return (
+    <ChartContainer
+      dataset={dataset}
+      loading={loading}
+      Placeholder={LineChartPlaceholder}
```

Comment:
I don't think a LineChartPlaceholder fits for a scatter chart - maybe create a new one?